versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -110,18 +131,68 @@ public long expireAfterRead(
                   singletonList(Tag.of("cache", CACHE_NAME)),
                   "",
                   x -> config.capacityMb());
+              meterRegistry.gauge(
+                  "cache_weight_tracked_mb",
+                  singletonList(Tag.of("cache", CACHE_NAME)),
+                  "",
+                  x -> (double) currentWeightTracked() / ONE_MB);
+              meterRegistry.gauge(
+                  "cache_weight_reported_mb",
+                  singletonList(Tag.of("cache", CACHE_NAME)),
+                  "",
+                  x -> (double) currentWeightReported() / ONE_MB);
+              meterRegistry.gauge(
+                  "cache_rejected_weight_mb",
```

Comment:
Why not a `DistributionSummary` similar to caffeine's evicted weight metrics?

I'm not sure a "guage" is correct here, because it's a cumulative value.

Reply:
True, a DS seems better here.

However, I also updates the other metrics to report the values as bytes, but also specify the base-unit `bytes` in the metric. AFAIK the base-unit's exposed so the metrics should also "magically" appear correctly.